include/PR/gbi.h

```diff
@@ -2161,6 +2167,26 @@ typedef union {
 	__gsSPLine3D_w1f(v0, v1, wd, flag)				\
 }}
 
+// Early F3DEX revisions use the actual quadrangle command
+// without a flag argument, this was added for compatibility.
```

Comment:
Could you please update this comment to be more specific. Perhaps along the lines of
```c
// mk64's startup logo uses an early revision of the quadrangle command
// It does not have a flag argument.
// Added for compatibility with F3DEX 0.95
```
The rest of the game uses a quad or tri2 which is equivallent to the newer f3dex versions.

Reply:
Both G_QUAD and G_TRI2 exist on `F3DEX 0.95`, while on 1.21 they both use `G_TRI2`. While it's possible they could have used quad on other display list, using tri2 makes them more consistent with already existing tri2 commands.